traffic_ops/v4-client/division.go

```diff
@@ -52,6 +52,17 @@ func (to *Session) GetDivisions(header http.Header) ([]tc.Division, toclientlib.
 	return data.Response, reqInf, err
 }
 
+// GetDivisions with Params returns all Divisions with matching params in Traffic Ops
+func (to *Session) GetDivisionsWithParams(header http.Header, params url.Values) ([]tc.Division, toclientlib.ReqInf, error) {
```

Comment:
Rather than create a new `GetDivisionsWithParams()` function, it might be a good idea to just add a `url.Values` parameters to the existing `GetDivisions()` function.

Changing the function signature is okay because `v4-client` is not yet included in any Apache Traffic Control release.

Reply:
removed GetDivisionsWithParams and replaced with GetDivisions, updated the function signature